Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -47,6 +47,9 @@
     @Column(name = "sortHelperImages")
     private Integer sortHelperImages;
 
+    @Column(name = "exportable")
+    private boolean exportable;
```

Comment:
You add property but no setter and getter for it... Hibernate will not work correctly in such a case.